docs/knowledge_base/lockfile-generation.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Lockfile Generation
+category: 6255e67693d5200013b1fa41
+hidden: false
+---
+
+The Phylum CLI can generate a lockfile when it is given a manifest file.
+
+### Lockfile generators
+
+| Lockfile type | Filenames        | Required tool               |
+| ------------- | ---------        | -------------               |
+| `npm`         | `package.json`   | [`npm`][npm]                |
+| `yarn`        | `package.json`   | [`yarn`][yarn]              |
+| `pip`         | `requirements*.txt` <br/> `requirements.in` <br/> `setup.py` <br/> `setup.cfg` <br/> `pyproject.toml` | `pip-compile` (from [`pip-tools`][pip-tools]) |
+| `pipenv`      | `Pipfile`        | [`pipenv`][pipenv]          |
+| `poetry`      | `pyproject.toml` | [`poetry`][poetry]          |
+| `gem`         | `Gemfile`        | `bundle` (from [Bundler][]) |
+| `mvn`         | `pom.xml`        | `mvn` (from [Maven][])      |
+| `gradle`      | `build.gradle`   | [`gradle`][gradle]          |
+| `go`          | `go.mod`         | [`go`][go]                  |
+| `cargo`       | `Cargo.toml`     | [`cargo`][cargo]            |
+
+[npm]: https://nodejs.org/
+[yarn]: https://yarnpkg.com/
+[pip-tools]: https://github.com/jazzband/pip-tools/
+[pipenv]: https://github.com/pypa/pipenv
```

Comment:
```suggestion
[pip-tools]: https://pip-tools.readthedocs.io/
[pipenv]: https://pipenv.pypa.io/
```
Pointing to the docs page instead of the repository is more in line with the other links here.

Reply:
The links chosen here were the "homepage" links for each package. See [pip-tools](https://pypi.org/project/pip-tools/) and [pipenv](https://pypi.org/project/pipenv/).